src/main/java/org/terasology/blockNetwork/NetworkNode.java

```diff
@@ -19,10 +19,29 @@
 import org.terasology.math.SideBitFlag;
 import org.terasology.math.geom.Vector3i;
 
+/**
+ * Represents a single node in a block network.
+ * Any instance of this class or its children can be used as a node in {@link EfficientNetwork}
+ */
 public class NetworkNode {
+    /**
+     * The location of the Node
+     */
     public final ImmutableBlockLocation location;
+    
+    /**
+     * Based on a combination of the input and output sides, i.e. any side that is an input, output, or both.
+     */
     public final byte connectionSides;
+    
+    /**
+     * Sides which can be used for input
+     */
```

Comment:
Comment.

Reply:
Scratch that, remove this line. Redundant information. You're converting `inputSides` to `Input Sides`. This is not the purpose of comments, or javadoc.